### ISSUE TITLE
[ScenarioDependencies] replaces DI implementation of the framework or the library(s)

### ISSUE THREAD

**Issue by viveksacademia4git** [solidtoken/SpecFlow.DependencyInjection#89]
### Background

I want to inject the dependencies for the scenarios. To do this I have followed the code from the [README](https://github.com/solidtoken/SpecFlow.DependencyInjection/blob/main/README.md#usage) and I performed the equivalent implementation, see implementation section:

Along with dependencies that I inject manually using the `[ScenarioDependencies]` attribute, I also wanted to have other dependencies like `specFlowOutputHelperHelper` available in the constructor as well; which are injected by the framework automatically in the background.

### Implementation

My implementation logic is as follows:

```
class Hook
{
  . . .
  [ScenarioDependencies]
  public static IServiceCollection CreateServices()
  {
    var services = new ServiceCollection();
    services.AddScoped<MyDependency>();
    return services;
  }
  . . .
}


class StepDefinition
{
  . . .
  public StepDefinition(MyDependency myDependency, 
        ISpecFlowOutputHelper specFlowOutputHelperHelper, ScenarioContext scenarioContext
        )
  {
     . . .
  }
  . . .
}
```

### Problem

The mechanism implemented in `[ScenarioDependencies]` somehow replaces a part of the DI implementation (auto-injection of dependencies) of the framework or the other library(s). For instance, I can no longer get an instance of `SpecFlowOutputHelper` class automatically in the constructor, within the binding step definition class i.e. `StepDefinition`. I get following error:
```
System.InvalidOperationException : Unable to resolve service for type 'TechTalk.SpecFlow.Infrastructure.ISpecFlowOutputHelper' while attempting to act...
```
To get the instances of these classes, I have to manually inject their objects.

### Question

How can we preserve the DI implementation of the framework or the library(s)?

**Comment by mbhoek** [comment]
Hello @viveksacademia4git, apologies for not getting back to you sooner. I believe this issue was fixed in v3.9.3 of the plugin (it automatically registers proxy bindings to the underlying DI framework (https://github.com/solidtoken/SpecFlow.DependencyInjection/blob/main/SpecFlow.DependencyInjection/DependencyInjectionPlugin.cs#L131). Please let me know it this solved your problem.

**Comment by viveksacademia4git** [comment]
Thank you for updating.
